docs/src/pages/careers/ROLE_TEMPLATE.md

```diff
@@ -37,7 +37,7 @@ XXXXXX
 
 XXXXXX
 
-Our solution empowers React developers to build awesome applications – hundreds of thousands of developers use MUI every month.
+Our solution empowers React developers to build awesome applications faster – we see over a million developers on MUI's docs every month. – hundreds of thousands of developers use MUI every month.
```

Comment:
```suggestion
Our solution empowers React developers to build awesome applications faster – we see over a million developers on MUI's docs every month. Hundreds of thousands of developers use MUI every month.
```

Reply:
Could we use "enables" in preference to "empowers"? It's less "business BS buzzwordy".